app/component/summary/OriginDestinationBar.js

```diff
@@ -23,12 +23,15 @@ class OriginDestinationBar extends React.Component {
     tabOpen: false,
   };
 
-  componentWillMount = () => {
+  componentWillMount() {
```

Comment:
IIRC while it wasn't unanimous, we decided that binding lambdas are the default whenever `this` doesn't explicitly need to be unbound, even if react handles the magic automagically for the lifecycle methods.